testing/go/information_schema_test.go

```diff
@@ -0,0 +1,41 @@
+package _go
+
+import (
+	"testing"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+func TestInfoSchemaTables(t *testing.T) {
+	RunScripts(t, []ScriptTest{
+		{
+			Name: "information_schema.tables",
+			SetUpScript: []string{
+				"create table test_table (id int)",
+			},
+			Assertions: []ScriptTestAssertion{
+				{
+					Query: `SELECT table_schema FROM information_schema.tables group by table_schema order by table_schema;`,
```

Comment:
Using group by is kind of a weird way to test this. Can you do distinct instead?